flexmeasures/api/dev/sensors.py

```diff
@@ -157,8 +159,18 @@ def get(self, id: int, asset: GenericAsset):
 
         .. :quickref: Chart; Download asset attributes for use in charts
         """
-        attributes = ["name", "timezone", "timerange_of_sensors_to_show"]
-        return {attr: getattr(asset, attr) for attr in attributes}
+        # attributes = ["name", "timezone", "timerange_of_sensors_to_show"]
+        # return {attr: getattr(asset, attr) for attr in attributes}
+        attributes = ["name", "timezone"]  # , "timerange_of_sensors_to_show"]
+        return {
+            **{attr: getattr(asset, attr) for attr in attributes},
+            **{
+                "timerange_of_sensors_to_show": {
+                    "start": datetime.datetime(2020, 12, 3, 14, 0, tzinfo=pytz.utc),
```

Comment:
This looks like a hard-coded value used in development?

Reply:
Indeed. I encountered significant page loading delay here. I'll make a new ticket to make this asynchronous.

Reply:
See #515.